ext/LoggingPolyglotExt.jl

```diff
@@ -2,10 +2,10 @@ module LoggingPolyglotExt
 
 using SimpleValidations, LoggingPolyglot
 
-function SimpleValidations.validation_error(;
-    collection::String = "",
-    attribute::String = "",
-    identifier::Union{String, Real} = "",
+function SimpleValidations.validation_error(
```

Comment:
We can keep it as keyword args but without the default value

Reply:
Kwargs with no default would not solve the issue.

But the function still works with kwargs: the method in the main package has kwargs just to call the methods with standard args.

The API does not change and no upstream code should require changes :)